R/nhs_navlistPanel.R

```diff
@@ -1,18 +1,27 @@
-#' nhs_navlistPanel Function
+#' nhs_navlistPanel
 #'
-#' @noRd
+#' NHS style navlist element
 #'
-#' @importFrom shiny tagList
+#' @inheritParams shiny::navlistPanel 
+#' 
+#' @return HTML
+#' @export
+#'
+#' @examples
+#' nhs_navlistPanel(
+#'   "nav_panel",
+#'   shiny::tabPanel(title = "Introduction", shiny::p("This is an intro")),
+#'   shiny::tabPanel(title = "Article", shiny::p("This is an article"))
+#' )
 nhs_navlistPanel <- function(...,
                              id = NULL,
                              selected = NULL,
                              header = NULL,
                              footer = NULL,
                              fluid = TRUE,
                              widths = c(4, 8)) {
-
   # Create navlist panel
-  nhs_navlistPanel <- shiny::navlistPanel(
```

Comment:
This is quite confusing to have an internal variable named the same as the function itself. I was running into some trouble and it made it difficult to troubleshoot!